Modules/Core/Mesh/include/itkTriangleMeshCurvatureCalculator.hxx

```diff
@@ -85,6 +85,7 @@ template <typename TInputMesh>
 void
 TriangleMeshCurvatureCalculator<TInputMesh>::ComputeGaussCurvature(MeshConstPointer inputMesh)
 {
+  MeshPointType v0, v1, v2;
```

Comment:
Instead of calling `Fill(0)` for each of these three variables, they may be initialized by an empty initializer-list, `{}`:

    MeshPointType v0{};
    MeshPointType v1{};
    MeshPointType v2{};